Build secure aws lambda nodejs react

reverentgeek opened the pull request:
This blog post has the following features:

- [x] A GitHub Repository with a polished README
- [x] A title that's approved by Dev Marketing
- [x] A URL that's approved by Dev Marketing
- [x] Rendered locally and confirmed that no Markdown typos exist

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+
+Next, set up AWS to handle your serverless operations.
+
+### Set up Lambda Functions
```
Should be consistent on how we spell setup - one word or two. Also, capitalization of Up if two.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+    return response;
+};
+
+```
````
Remove unnecessary line break. Also, I think JS uses 2 space indentation more often than 4.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+
+```
+
+AWS provides a simple blank, "Hello World," output with a response code of 200. Keep that code, but replace your body with the `getCards` function call. `getCards` calls `buildDeck` which creates your 52 cards—ace through King of each suit. In this game, aces will be considered low, making their value 1. If you wish to make them high, change the value to 14. For an added challenge, consider how y ...
````
I typically add () after function names.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+
+AWS provides a simple blank, "Hello World," output with a response code of 200. Keep that code, but replace your body with the `getCards` function call. `getCards` calls `buildDeck` which creates your 52 cards—ace through King of each suit. In this game, aces will be considered low, making their value 1. If you wish to make them high, change the value to 14. For an added challenge, consider how y ...
+
+Next, you need to test your function. This is relatively easy—click the *Test* button on the top of your screen. You should then see  a screen that says "Configure Test Event" with some JSON dummy values in it. These values will be passed to your function in the `event` object. Since building the deck doesn't require any input, feel free to leave this as an empty JSON object. Name your test trigg ...
```
I usually bold links and buttons that users click on, rather than italics.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+
+    const getHighScores = () => {
+
+        var scores = [];
```
Why var when const will work?

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+
+## The React Application
+
+With your API set up and ready to go, you can begin to work on the react.js application.
```
Capitalize React.

mraible replied:
Here and below.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+
+After clicking deploy, you should land on the *Stages* page on your API. You can open any of the stages you have created and select the `/deck/get` method. There you will see an alert with the text *Invoke URL: {some url}*. This URL is what your application will need to call.
+
+## The React Application
```
This title can be better, IMO.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+
+With your API set up and ready to go, you can begin to work on the react.js application.
+
+### Setup your Application
```
Your should be capitalized and stuffing a keyword in here can be good for SEO.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+npm install @okta/okta-signin-widget@3.9
+```
+
+Next, use bootstrap for styling. To do this, install *React-Bootstrap* as well as *Bootstrap.*  *React-Bootstrap* is used to bridge the gap between react.js and bootstrap, however, it does not ship with the bootstrap CSS so you will need to install that as well.
````
Bootstrap should be capitalized.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+
+In pages, add the following files:  `Game.jsx`, `Home.jsx`, `Login.jsx`. You can now work on implementing each of these pages.
+
+### Implement the Components
```
Add React for keywords and SEO.

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+
+First you can implement the `AppWithRouterAccess.jsx` file.
+
+```javascript
````
Change to jsx

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+
+Now, shift your focus to the `Components` folder. Start by setting up your `Header` component.
+
+```javascript
````
Jsx

mraible replied:
Be consistent with indents, should be 2.

mraible replied:
Need raw tags because {{...}}

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+
+Next, implement the ``LoginForm`` component.
+
+```javascript
````
jsx

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+export default LoginForm;
+```
+
+When the user submits the form, your application will override that action and direct the user into the Okta workflow. This is done with ``oktaAuth.signIn`` which returns a promise that must be handled appropriately. If the user has successfully logged in, set the `sessionToken` that allows the user to proceed authenticated.
````
Why double back tics?

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+
+Let's start with the `Card` component that will be used in the game.
+
+```javascript
````
jsx and two spaces

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
```diff
@@ -0,0 +1,1057 @@
+If the card is listed as visible, you will display the card rank and suit based on the CSS provided by the `CSS-Playing-Cards` library. If it is not, show the back of the card.
+
+With that done you can work on the `GameHome` component.
+
```
jsx and two spaces for this one + remaining code blocks

mraible commented on _source/_posts/2020-04-22-build-secure-aws-lambda-nodejs-react.md:
````diff
@@ -0,0 +1,1057 @@
+export default App;
+```
+
+## Test it
````
This header and the one below could use more keywords since they're h2s

reverentgeek commented:
@mraible Thanks for the thorough review! I think I've addressed everything. I ran all the code through a linter/formatter to clean everything up. Found a couple of issues in the process. Also fixed some other consistency issues.
